spec/models/user_spec.rb

```diff
@@ -0,0 +1,18 @@
+RSpec.describe User, type: :model do
+
+    describe 'DB table' do
+        it { is_expected.to have_db_column :name }
+    end
+
+    describe 'Validations' do
+        it { is_expected.to validate_uniqueness_of :name }
+        it { is_expected.to validate_presence_of :email }
+        it { is_expected.to validate_presence_of :password }
+    end
+
+    describe 'Factory' do
+        it 'expects a valid Factory' do
+            expect(FactoryBot.create(:user)).to be_valid
+        end
+    end
+end
```

Comment:
Just because you made it looks so nice with 2 spaces indentation in the other file, please fix it here also 😉